lib/wikidata/fetcher.rb

```diff
@@ -7,10 +7,16 @@
 
 class WikiData
   class Fetcher < WikiData
+    LOOKUP_FILE = 'https://raw.githubusercontent.com/everypolitician/wikidata-fetcher/master/lookup.json'.freeze
+
     def self.find(ids)
       Hash[Wikisnakker::Item.find(ids).map { |item| [item.id, new(item: item)] }]
     end
 
+    def self.lookup
+      @lookup ||= JSON.parse(open(LOOKUP_FILE).read, symbolize_names: true)
```

Comment:
Yep good point. I've renamed it `wikidata_properties` in https://github.com/everypolitician/wikidata-fetcher/pull/35/commits/b43a7d4b4f69ea4a73a527eb670934eec2f59e4b as that makes code like `wikidata_properties[:want]` and `wikidata_properties[:skip]` make a bit more sense.